core/kover/dataset/create.py

```diff
@@ -286,8 +317,10 @@ def normal_raise(exception):
 
     # Extract/write the metadata
     if phenotype_name is not None:
```

Comment:
We might want to change the concept of phenotype name to a "description" in the command line interface and in the created dataset. This would be more general and encompass both the binary and multiclass settings.